app/src/main/res/drawable/ic_file_hidden.xml

```diff
@@ -0,0 +1,8 @@
+<!-- drawable/file_hidden.xml -->
```

Comment:
Please remove this comment and add the copyright notice on all new files (see ic_sidenav_settings.xml)